### ISSUE TITLE
Support Cred computation intervals other than weekly

### ISSUE THREAD

**Issue by decentralion** [sourcecred/sourcecred#2449]
It'd be nice to add support for calculating Cred on a finer-grained time resolution, e.g. daily. Creating this issue mostly to add linked comments in the codebase.

**Comment by elihanover** [comment]
I agree. I was looking at the `GrainConfig` process and thought that it would be worthwhile at least decoupling time from the policy type.

For example in an instance's `config/grain.json`
```js
// config/grain.json
{
  "IMMEDIATE": {
      "weekly": 100,
  },
  "BALANCED":  {
      "monthly": 1000 
  },
  "maxSimultaneousDistributions": 100
}
```
I'm happy to get rolling on this if you think it's the right first step.
